server/core/src/irods_report_plugins_in_json.cpp

```diff
@@ -145,6 +166,17 @@ namespace irods {
             return PASS(ret);
         }
 
+        std::string comments;
+        ret = _resc->get_property< std::string >( irods::RESOURCE_COMMENTS, comments );
+        if (!ret.ok()) {
+            return PASS(ret);
+        }
+        std::string info;
+        ret = _resc->get_property< std::string >( irods::RESOURCE_INFO, info );
+        if (!ret.ok()) {
+            return PASS(ret);
+        }
```

Comment:
You could reuse the `std::string` for both of these properties if `_entry` is updated once the value is retrieved.

Obviously, not incorrect, but as-is, this code allocates more memory than necessary.

Reply:
I agree, one string could be used instead of multiple. Should I use one string for all the other fields or restrict it to just comments and info?

Reply:
Let's focus on just these two fields for now.